front/cypress/e2e/survey_builder/create_and_fill_survey.cy.ts

```diff
@@ -508,14 +458,11 @@ describe.skip('Survey builder', () => {
     cy.get('[data-cy="e2e-field-rule-display"]')
       .contains(chooseOneOption1)
       .should('exist');
-    cy.get('[data-cy="e2e-field-rule-display"]')
-      .contains('Survey end')
-      .should('exist');
     cy.get('[data-cy="e2e-field-rule-display"]')
       .contains(chooseOneOption2)
       .should('exist');
     cy.get('[data-cy="e2e-field-rule-display"]')
-      .contains('Survey end')
+      .contains('Ending')
```

Comment:
I was wondering if it's a good idea to have tests depend on translations (which can be changed in Crowdin).

Reply:
Usually no - if we do use them, it's something because the test was flaky otherwise :) But in general, if it works using a data-cy or ID instead, that's indeed preferable :+1:

Reply:
I tried to use the Crowdin translation but didn't succeed. I'll make another attempt, and if I remain stuck, I'll just merge like this. I think I was mostly struggling with getting the translation from the right translation file.